packages/sdk/package.json

```diff
@@ -36,19 +36,19 @@
   "dependencies": {
     "@commercetools-frontend/actions-global": "2.0.0",
     "@commercetools-frontend/constants": "2.0.0",
-    "@commercetools/api-request-builder": "4.1.0",
-    "@commercetools/sdk-client": "1.5.9",
+    "@commercetools/api-request-builder": "4.1.1",
+    "@commercetools/sdk-client": "1.5.10",
     "@commercetools/sdk-middleware-correlation-id": "1.0.1",
-    "@commercetools/sdk-middleware-http": "4.0.7",
-    "@commercetools/sdk-middleware-user-agent": "1.2.10",
+    "@commercetools/sdk-middleware-http": "4.0.8",
+    "@commercetools/sdk-middleware-user-agent": "1.2.11",
```

Comment:
This are already likely double bundled I presume due to how webpack will resolve.